README.md

```diff
@@ -1,23 +1,145 @@
-# ttoss Monorepo Template
+<h1 align="center">
 
-Monorepo template used by [Triãngulos Tecnologia](https://triangulostecnologia.com) and ttoss projects.
+`LPS - Landing Pages`
 
-## Features
+</h1>
 
-- Eslint
-- Commitlint
-- Husky
-- Jest
-- Lint Staged
-- NPM Check Updates
-- Prettier
-- Stylelint
-- TypeScript
+<p align="center">
+<a href="https://github.com/TriangulosTecnologia/lps/blob/main/LICENSE"></a>
+    <img src="https://img.shields.io/badge/license-MIT-green" alt="License" />
+  </a>
+</p>
 
-## Init Husky
+<br/>
+
+## 📚 About
+
+<strong> Landing Page</strong> is a page that has all the elements aimed at converting the visitor into Lead, opportunity or customer. Also as landing pages or conversion pages, Landing Pages are widely used in Digital Marketing campaigns, as they tend to have high conversion rates.
```

Comment:
Landing Page is a website with all the elements to convert visitors into leads, opportunities, or customers. Also, like landing pages or conversion pages, Landing Pages are widely used in Digital Marketing campaigns, as they tend to have high conversion rates.